appvm-scripts/usrbin/qubes-run-xorg.sh

```diff
@@ -69,4 +69,4 @@ chmod 770 /var/run/xf86-qubes-socket
 
 export XDG_SEAT=seat0 XDG_VTNR=7 XDG_SESSION_CLASS=user
 
-exec su -l user -c "/usr/bin/xinit $XSESSION -- $XORG :0 -nolisten tcp vt07 -wr -config xorg-qubes.conf > ~/.xsession-errors 2>&1"
+exec su -l $(qubesdb-read /default-user) -c "/usr/bin/xinit $XSESSION -- $XORG :0 -nolisten tcp vt07 -wr -config xorg-qubes.conf > ~/.xsession-errors 2>&1"
```

Comment:
This will break badly if user first update template and only then dom0. It should default to `user` if `/default-user` is missing.